_dev/src/components/smart-shopping-campaign/smart-shopping-campaign-table-list.vue

```diff
@@ -313,9 +313,6 @@ export default {
         });
     },
     handleScroll() {
-      if (this.loading === true) {
-        return;
-      }
```

Comment:
This check prevents to trigger several loading of the next page at the same time.